### PR TITLE
Disable default SLE repo

### PR DESCRIPTION
**Pull request by dzedro**
Fix [failing](https://openqa.suse.de/tests/793214#step/zypper_lr/12) post registration via YaST

[test](http://10.100.12.155/tests/5495)

### CODE REVIEW

File: tests/installation/addon_products_via_SCC_yast2.pm [hunk 1]
```diff
@@ -26,19 +26,20 @@ sub run() {
             push(@addon_proxy, "\b.$addon-" . get_var("BUILD_$uc_addon"));
         }
         script_sudo "echo \"@addon_proxy.proxy.scc.suse.de\" > /etc/SUSEConnect", 0;
+        script_sudo 'zypper mr -d 1',                                             0;    # zypper_lr test is expecting this repo disabled
```
Comment (sysrich) [inline tests/installation/addon_products_via_SCC_yast2.pm]: can you guarantee that repo #1 is ALWAYS going to be the one you want to disable? Seeing a magic number ilke that makes me nervous

**Comment by dzedro** [comment]
I don't know why is zypper_lr test expecting disabled iso repo

**Comment by sysrich** [comment]
Ask the author of sub `validate_repos` and `validate_repos_sle` in lib/utils then?

I think it's a lot more likely that we have a bug in the zypper_lr test than all of our customers need to disable their installation repo manually after installation for registration to work..

File: tests/installation/addon_products_via_SCC_yast2.pm (hunk 1 above)
Comment (dzedro) [inline tests/installation/addon_products_via_SCC_yast2.pm]: true, it will be one of the first repos, but not always first

**Comment by dzedro** [comment]
I will open bug, it is not clear how it should be when system is registered after registration.

**Comment by dzedro** [comment]
from fate#320494

**Comment by dzedro** [comment]
https://bugzilla.suse.com/show_bug.cgi?id=1027613

**Pull request closed without merging**